### ISSUE TITLE
Connection pool does not create when initializing cluster has an offline node

### ISSUE THREAD

**Issue by tnine** [s7/scale7-pelops#24]
To reproduce this issue.  Perform the following.
1. Start testing cluster.
2. Create a cluster, any keyspace, with timeout.  Use nodes 127.0.0.2, and 127.0.0.1. 
3. Initialize the pool.

The initialization hangs because 127.0.0.2 is not online, which is correct.  However, the pool should continue to initialize.

**Comment by tnine** [comment]
Correction.  The pool does initialize, however the timeout is not equal to the timeout I've set in my Config object.  I have set my timeout to 10000, however it appears to use the default 60000 instead.

**Comment by danwashusen** [comment]
Hi Todd,
I've just added CommonsBackedPoolIntegrationTest#testInitWithDownedNode to try and verify the issue but it appears to be working (as stated in your original issue and in your comment).  Would you be able to take a look at the test and verify?

Also, the pool currently blocks while attempting to create a connection to the missing node.  This creates a pause of the timeout value for each node that can't be contacted.  The reason for the delay is Pelops is trying to create enough connections to satisfy the minIdlePerNode config property.  We could delay this until the node is first access but then that request will wear the timeout...  An alternative would be to set the minIdlePerNode to zero which essentially achieves the same thing.

Thoughts?

Cheers,
Dan

**Comment by YanivKunda** [comment]
testInitWithDownedNode fails for me - but using some enhancements I've made (available in https://github.com/s7/scale7-pelops/pull/25) I found that my effective timeout is ~1s -
the connection to 127.0.0.2 results in a connection refused drop, rather than an actual timeout.
When I replaced to an address valid in my subnet, but not connected to any NIC, the test passed.
Maybe we need to find an address that is universally unavailable, or perhaps "compute" it (i.e. scan for one).
